### ISSUE TITLE
[CrC] [Question] Aura Tracker (or any Powered Tool) Charging

### ISSUE THREAD

**Issue by chrisfirefox** [ReikaKalseki/Reika_Mods_Issues#2280]
Hey there, I tried to find out myself, I even delved into the sourcecode on this one, but I just can't seem to figure out how to charge the Aura Tracker. IMO and according to the code, I should be able to charge it be getting within 32m of a black (=Kuro?) Energy Pylon. I'm still in early game, so this probably would work getting close to a black Lumen Repeater later on, but that is afaik more down the line in progression.

So, any Idea what I'm doing wrong? Or did I just misinterpret the code completely? Because as far as I read it, charging needs to have the item to be able to charge (charge < getMaxCharge), have it active (which is equal to "held") and have a ConductingElement(Black) in range - which should apply to pylons, I wasn't able to verify that.

I verified the distance by trying to draw with the Manipulator and was able to do so. Again, any suggestions would be highly appreciated, thank you!

**Comment by Barhandar** [comment]
You throw it on the ground next to a pylon, preferably into a pool of chroma.

"Active" means "loaded and is in the list for ticking", not "held".

**Comment by chrisfirefox** [comment]
https://github.com/ReikaKalseki/ChromatiCraft/blob/db280df000eb88a4e56ba4a21328a74cf293d58f/Items/Tools/Powered/ItemStructureFinder.java#L177 says that "isActivated" is equal to "has Tool in hand". I already tried throwing it on the ground, the code also states that there should be a particle effect. But I'll gladly try it again with a pool of chroma. Will report back when I did, just can't at this moment :).

Edit: My bad, isActivated is relevant for USAGE, not for charging. After rereading the code for onEntityUpdate, this should actually work, and chroma would increase the recharge rate by 1.25. Well, I'll try again :)
